core/src/main/java/org/kiwix/kiwixmobile/core/utils/Constants.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2020 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+package org.kiwix.kiwixmobile.core.utils
+
+object Constants {
```

Comment:
We don't need an object to store constants. You can declare them directly in the file.

Reply:
@abdulwd If I store constants without any class/object, I would need to it import it as
`import org.kiwix.kiwixmobile.core.utils.VARIABLE_NAME` There would be no use of using "Constants" as the filename. I thought using `object` will make the code more clear.

If using `org.kiwix.kiwixmobile.core.utils.VARIABLE_NAME` seems okay, I can make the changes.

Reply:
I rarely check how's the variable imported. I use code navigation `Command + Click` to jump to the definition.

Reply:
@abdulwd I have made the required changes, please check.